index.bs

```diff
@@ -6124,6 +6314,22 @@ partial dictionary MLOpSupportLimits {
     **Returns:** an {{MLOperand}}. The reduced output tensor. If the input operand is a scalar, the reduction function is applied to the scalar value, and the output is also a scalar.
 </div>
 
+<table id=constraints-reduction class='data' link-for="MLGraphBuilder/reduceL1(input, options), MLGraphBuilder/reduceL2(input, options), MLGraphBuilder/reduceLogSum(input, options), MLGraphBuilder/reduceLogSumExp(input, options), MLGraphBuilder/reduceMax(input, options), MLGraphBuilder/reduceMean(input, options), MLGraphBuilder/reduceMin(input, options), MLGraphBuilder/reduceProduct(input, options), MLGraphBuilder/reduceSum(input, options), MLGraphBuilder/reduceSumSquare(input, options)">
+  <caption>Constraints for reduction-operations</caption>
+  <thead>
+    <tr>
+      <th>input operand</th>
+      <th>[=/allowed data types=]</th>
+      <th>[=/allowed ranks=]</th>
+    </tr>
+  </thead>
+  <tr>
+    <td>{{input}}</td>
+    <td>[=/any data type|any=]</td>
```

Comment:
"specified as part of operation steps"? For example `reduceLogSum` only takes floating-point input.

Reply:
Ditto!

Reply:
Fixed in db8d1e21ef42e7897fd7143d41dea07194dee96c